singularity-eos/closure/mixed_cell_models.hpp

```diff
@@ -259,15 +272,11 @@ class PTESolverBase {
   // after each iteration of the Newton solver.  This version just renormalizes the
   // volume fractions, which is useful to deal with roundoff error.
   PORTABLE_INLINE_FUNCTION
-  virtual void Fixup() const {
-    Real vsum = sum_neumaier(vfrac, nmat);
-    for (std::size_t m = 0; m < nmat; ++m)
-      vfrac[m] *= robust::ratio(vfrac_total, vsum);
-  }
+  virtual void Fixup() { NormalizeVfrac(); }
   // Finalize restores the temperatures, energies, and pressures to unscaled values from
   // the internally scaled quantities used by the solvers
   PORTABLE_INLINE_FUNCTION
-  void Finalize() {
+  virtual void Finalize() {
```

Comment:
We're testing this on GPU, so I guess this works, right?

Reply:
I don't think we provide overrides to these implementations. My preference is to remove the keyword if it is not needed.